src/Home/Home.scss

```diff
@@ -0,0 +1,67 @@
+
+
+.postOTDContainer{
+    font-family: "Segoe UI";
+    color: #DCDCDC;
+    position: relative;
+    width: 90%;
+    left: 5%;
+    padding-left: 0;
+    background-color: #8E6A5A;
+    margin-top: 20%;
+    margin-bottom: 5%;
+    padding-bottom: 3px;
+    border-radius: 25px;
+}
+.weatherTable{
+    position: relative;
+    width: 90%;
+    left: 5%;
+    background: none;
+    margin-bottom: 20%;
+    border-radius: 25px;
+    
+    
+}
+.postOTD{
+    width: 100%;
+    
+}
+.postTitle{
+    position: absolute;
+    left: 2%;
```

Comment:
A lot of the time we use px/rem to do spacing to keep it consistent instead of percentages, width:90% and things like that are all good but I think for these small left/margins/padding values we want to use set values in px